README.md

```diff
@@ -0,0 +1,38 @@
+# create-svelte
```

Comment:
Do we need this file?

Reply:
The readme will eventually include instructions to make edits to the website!